src/status_im2/contexts/communities/overview/view.cljs

```diff
@@ -57,48 +71,28 @@
                     (int (Math/ceil (layout-y %))))
                  (into #{} (map (comp :name second) channels-list)))
     :style     {:margin-top 20 :flex 1}}
-   (for [[category-id {:keys [chats name collapsed?]}] channels-list]
-     [rn/view
-      {:flex      1
-       :key       category-id
-       ;; on-layout fires only when the component re-renders, so
-       ;; in case the category hasn't changed, it will not be fired
-       :on-layout #(on-category-layout name (int (layout-y %)))}
-      (when-not (= constants/empty-category-id category-id)
-        [quo/divider-label
-         {:label            name
-          :on-press         #(collapse-category
-                              community-id
-                              category-id
-                              collapsed?)
-          :chevron-icon     (if collapsed? :main-icons/chevron-right :main-icons/chevron-down)
-          :padding-bottom   (if collapsed? 7 0)
-          :chevron-position :left}])
-      (when-not collapsed?
-        [rn/view
-         {:style {:margin-left   8
-                  :margin-top    10
-                  :margin-bottom 8}}
-         (for [{:keys [muted? id] :as chat} chats
-               :let                         [chat           (assoc chat
-                                                                   :chat-type
-                                                                   constants/community-chat-type)
-                                             channel-muted? (or muted?
-                                                                (rf/sub [:chat/check-channel-muted?
-                                                                         community-id id]))]]
-           [rn/view
-            {:key   (:id chat)
-             :style {:margin-top 4}}
-            [quo/channel-list-item
-             (assoc chat
-                    :muted?
-                    channel-muted?
-                    :on-long-press
-                    (fn []
-                      (rf/dispatch [:show-bottom-sheet
-                                    {:content       (fn [] [actions/chat-actions chat false])
-                                     :selected-item (fn []
-                                                      [quo/channel-list-item chat])}])))]])])])])
+   (doall
+    (for [[category-id {:keys [chats name collapsed?]}] channels-list]
+      [rn/view
+       {:style     {:flex 1}
+        :key       category-id
+        ;; on-layout fires only when the component re-renders, so
+        ;; in case the category hasn't changed, it will not be fired
+        :on-layout #(on-category-layout name (int (layout-y %)))}
+       (when-not (= constants/empty-category-id category-id)
+         [quo/divider-label
+          {:container-style  {:padding-left   16
+                              :padding-right  20
+                              :padding-top    6 ; Because of border width of 1
+                              :padding-bottom 7}
```

Comment:
Divider label overwriten styles, this won't be required when  #16462 is addressed.